cypress/integration/podcasts.spec.js

```diff
@@ -151,6 +151,29 @@ describe("newsdesk logged in", () => {
     cy.get("section[slotno=7] button[title='Delete']").click()
     cy.get("section[slotno=7]").should("not.contain", textToCopy)
   })
+
+  it("should add links in items", () => {
+    cy.contains("dev sandbox").click()
+    const link =
+      "https://twitter.com/PoliticusSarah/status/1520759587128979458?s=20&t=-ZSrWH2DIXO97dJgtoy46Q"
+    cy.get("section[slotno=7] textarea").type(`${link}{enter}`, {
+      force: true,
+    })
+    cy.get("section[slotno=7]").should(
+      "contain",
+      link.length > 50 ? link.slice(0, 50) + "..." : link,
```

Comment:
the link is a fixed variable, so i'm not sure why you're doing any conditional here
better would be
```suggestion
      "https://twitter.com/PoliticusSarah/status/1520759..."
```